### DISCUSSION TITLE
Zero Chia in May vs. Zero Chia in June

### DISCUSSION THREAD

**Discussion by FSys77** [Chia-Network/chia-blockchain#5858] [Show and tell]
![zeri chia final](https://user-images.githubusercontent.com/81809431/119254302-de6b9800-bbb5-11eb-9926-0b7186da05ab.png)

**Comment by FSys77** [comment]
The Top-10 mistakes of the Chia team:
1. Go live without pooling protocol
2. Go live without pooling protocol
3. Go live without pooling protocol
4. Go live without pooling protocol
5. Go live without pooling protocol
6. Go live without pooling protocol
7. Go live without pooling protocol
8. Go live without pooling protocol
9. Go live without pooling protocol
10. Go live without pooling protocol

**Reply by nonogrub** [reply]
This is the most botched Cryptocurrency project of all.

**Reply by room101-dev** [reply]
> The Top-10 mistakes of the Chia team:
> 
> 1. Go live without pooling protocol
> 2. Go live without pooling protocol
> 3. Go live without pooling protocol
> 4. Go live without pooling protocol
> 5. Go live without pooling protocol
> 6. Go live without pooling protocol
> 7. Go live without pooling protocol
> 8. Go live without pooling protocol
> 9. Go live without pooling protocol
> 10. Go live without pooling protocol

Come on with the truth,you can call this incompetence, or criminality. I choose to call it what it is, we already know that Bram&Hoffman have lied all along, they have admitted so to Gerard in a recent discussion, where they were forced to show their cards.

HPOOL is owned by CHINA, CCP that high. CHIA is owned by CCP, that's why the IOU's went to $1600

The entire deal is that China needed to have 100% control of CHIA in order to approve the deal, to fund the IOU's and to make Chia 'legal', while Bitcoin will become illegal to mine.

Chinese didn't want people in the West to hold XCH, so they were told to solo-mine, where nobody ever earns anything. Everybody in China ( asia for that matter ) have been mining chia all along since January 2021, 4 months ago Hong-Kong ran out of Hard-Disks, today Thailand has ran out of hard-disks, and most of the worlds hard-disks are made in Thailand. Funny thing is you can still buy Hard-Disks from Amazon in USA, that's the only place that hasn't ran out.

So in summary, they told you Western morons "Not to mine Chia on Hpool, we know that the insiders at Chia-Net have been running on the hpool all along", many even surmise that HPOOL was given the RPC interface early on, as early as last year to seal the deal.

CHINA was the investor, they own CHIA, Bram&Hoffman get to take the corp public as an IPO in USA, the corporation also pre-mined 21M xch. So they can sell these to buy jet-airplanes and yachts as needed.

The only funny thing in this entire scam, is how the woke morons, went along entire time and didn't mine on hpool. I myself months ago when I started plotting, the first thing I did was setup an HPOOL account, because I knew from years in this hobby that solo-mining is impossible. Today I have lots of XCH from hpool.

The most important point here is  that king of porn mr. bittorrent has already admitted publicly that he lied promoting chia all along, lied about everything. Why would he care about hurting little 'woke' western morons? Why? They're just ant's China is the owner of CHIA. 

The sheep are all herded in the West, mostly USA. People here who bitch about getting zero, are still waiting with their thumbs up their arse, waiting to be gifted by Chia-Net Dev, Inc. Insanity knows no bounds, the entire 'green' deal was to bring in new suckers to the crypto universe, it worked the marketing, and paid influencers on youtube brought in 10's of 1,000's of morons.

Lastly, very important, NO crypto on earth provides the "POOL", everybody provides an interface to 'get-new-template()" RPC so that the pool can create new blocks on every cycle. The fact that CHIA-NET only released this interface to HPOOL shows that they intended from day one, that HPOOL (CHINA-CCP) was to own CHIA forever.

**Reply by qpirsel** [reply]
_The fact that CHIA-NET only released this interface to HPOOL..._

Where does this information come from? Until now I thought hpool uses a proprietary backend with the offical CHIA client(s) in front of it.

**Reply by padik11** [reply]
Will you describe how you can hook up to a hpool? They do not have a detailed description on the website and the screenshots are not displayed for them

**Reply by joelrb** [reply]
blah blah blah blah blah

**Comment by playzhou** [comment]
hahaha

**Comment by yfycool** [comment]
You can go to hpool
You have zero income anyway
What are you afraid of losing?
